build.xml

```diff
@@ -200,10 +200,50 @@
 
     <!-- Jitsi Videobridge Version -->
     <target name="version">
+        <!-- extracts major version -->

```

Comment:
so this is still using ant, is that right? is there a maven method to achieve this? or even leverage a bash script or something maybe?  it'd be nice not to add new dependencies on ant.

Reply:
There is no new dependency added. We can do that but it is a separate project, not part of this one. All our java packages use ant. Ant and packaging is used even before maven was introduced for the java projects.

Reply:
i get that, but we'd prefer to be off of ant, no?  by 'new dependency' i didn't mean adding a dependency on ant entirely, but adding another 'feature' which relies on ant.  and i agree we shouldn't try and get off of ant now (can be a separate project, as you said) i just wondered if there was an approach for this that didn't use ant that wouldn't be so hard to do now.

Reply:
I don't want to do that now as it brings a lot of changes. And we will slow down stuff for a week or two which we cannot afford at the moment.  
Yes sure we can get rid of ant at some point, but this was not discussed and not planned at this point. This is the case for jicofo, jigasi and videobridge.
Half of the build process currently that is available for jvb can be ripped off as it is for building a Debian source package, which no longer works and is not something we will do in the future. 
But still, there is much more other stuff that is handled while building the package.